docs/topics/Gateway.md

```diff
@@ -96,8 +97,8 @@ def on_websocket_message(msg):
 ###### Gateway URL Params
 
 | Field     | Type    | Description                                   | Accepted Values                                                            |
-| --------- | ------- | --------------------------------------------- | -------------------------------------------------------------------------- |
-| v         | integer | Gateway Version to use                        | 6 (see [Gateway versions](#DOCS_TOPICS_GATEWAY/gateways-gateway-versions)) |
+|-----------|---------|-----------------------------------------------|----------------------------------------------------------------------------|
+| v         | integer | Gateway Version to use                        | see [Gateway versions](#DOCS_TOPICS_GATEWAY/gateways-gateway-versions) |
 | encoding  | string  | The encoding of received gateway packets      | 'json' or 'etf'                                                            |
 | compress? | string  | The (optional) compression of gateway packets | 'zlib-stream'                                                              |
```

Comment:
fix table spacing

Reply:
My autoformatter is really messing me up. I'm just gonna turn it off for now and do a separate pass in the future for spacing.